tests/test_encode_decode.py

```diff
@@ -95,8 +96,80 @@ def test_b85_encode_decode_round_trip(self, payload, pad):
 
 
 class TestBinASCII(unittest.TestCase):
-    # TODO: https://docs.python.org/3/library/binascii.html
-    pass
+    @given(payload=st.binary(), backtick=st.booleans())
+    def test_b2a_uu_a2b_uu_round_trip(self, payload, backtick):
+        x = binascii.b2a_uu(payload, backtick=backtick)
+        self.assertEqual(payload, binascii.a2b_uu(x))
+
+    @given(payload=st.binary(), newline=st.booleans())
+    def test_b2a_base64_a2b_base64_round_trip(self, payload, newline):
+        x = binascii.b2a_base64(payload, newline=newline)
+        self.assertEqual(payload, binascii.a2b_base64(x))
+
+    @given(
+        payload=st.binary(),
+        quotetabs=st.booleans(),
+        istext=st.booleans(),
+        header=st.booleans(),
+    )
+    def test_b2a_qp_a2b_qp_round_trip(self, payload, quotetabs, istext, header):
+        x = binascii.b2a_qp(payload, quotetabs=quotetabs, istext=istext, header=header)
+        self.assertEqual(payload, binascii.a2b_qp(x, header=header))
+
+    @given(payload=st.binary())
+    def test_rlecode_hqx_rledecode_hqx_round_trip(self, payload):
+        x = binascii.rlecode_hqx(payload)
+        self.assertEqual(payload, binascii.rledecode_hqx(x))
+
+    @given(payload=st.binary())
+    def test_b2a_hqx_a2b_hqx_round_trip(self, payload):
+        # assuming len(payload) as 3, since it throws exception: binascii.Incomplete, when length is not a multiple of 3
+        assume(len(payload) % 3 == 0)
```

Comment:
Let's actually test that:

```suggestion
        if len(payload) % 3:
            with self.assertRaises(binascii.Incomplete):
                binascii.b2a_hqx(payload)
            payload += b"\x00" * (len(payload) % 3)
```